opm/simulators/wells/GasLiftSingleWellGeneric.hpp

```diff
@@ -180,7 +184,7 @@ class GasLiftSingleWellGeneric : public GasLiftCommon<Scalar, IndexTraits>
             }
         }
 
-        Scalar oil, gas, water;
+        Scalar oil, gas, water, bhp;
```

Comment:
Refer to earlier comment about the name `BasicRates` and including `bhp` as a member.